channeldb/graph.go

```diff
@@ -2132,6 +2147,21 @@ func (c *ChannelGraph) FilterKnownChanIDs(chansInfo []ChannelUpdateInfo,
 			scid := info.ShortChannelID.ToUint64()
 			byteOrder.PutUint64(cidBytes[:], scid)
 
+			// Check that none of the timestamp is expired.
+			if isStale(info.Node1UpdateTimestamp) ||
```

Comment:
maybe this is too strict ? wdyt ?

Reply:
I think it's fine - however. i think maybe we should do these checks outside of the DB transaction. could be that after this filter, no DB tx is required

Reply:
ah - hmmm ok maybe we want to be slightly more relaxed and only skip if updates from _both_ nodes are out of date? 

if one node is sending updates it could be useful to get that latest update from them potentially... 

interested to hear what others think

Reply:
changed it so that both edges need to be out of date.